hack/dind-cluster.sh

```diff
@@ -212,17 +284,25 @@ function copy-runtime() {
   cp "${osdn_plugin_path}/sdn-cni-plugin/80-openshift-sdn.conf" "${target}"
 }
 
+function wait-for-master() {
+  local config_root=$1
+
+  local kubeconfig="$(get-admin-config "${config_root}")"
```

Comment:
If you mix scoping statements (`local`, `export`, `readonly`, `declare`, etc) and other constructs with return codes (like subshells) the return code of the scoping statement will alias that of the expression and if the expression fails this line will still succeed. So, we try to break them up:

```sh
local kubeconfig; kubeconfig="$( get-admin-config "${config_root}" )"
```